README.md

```diff
@@ -63,51 +63,53 @@ Our default export contains all of our Stylelint rules, along with specific plug
 | [`function-calc-no-unspaced-operator`][16]                   | Enabled                                                             |
 | [`function-linear-gradient-no-nonstandard-direction`][17]    | Enabled                                                             |
 | [`function-url-quotes`][18]                                  | `"always"`                                                          |
-| [`keyframe-declaration-no-important`][19]                    | Enabled                                                             |
```

Comment:
It’s interesting how useless the plain-text diff is here, while the Markdown rendering of the diff is very simple to review.